src/main/java/org/itsallcode/openfasttrace/report/PlainTextReport.java

```diff
@@ -130,54 +133,55 @@ private String translateStatus(final boolean ok)
         return ok ? "ok" : "not ok";
     }
 
-    // [impl->dsn~reporting.plain-text.summary~1]
+    // [impl->dsn~reporting.plain-text.summary~2]
     private void renderSummary(final PrintStream report)
     {
-        report.print(translateStatus(this.trace.isAllCovered()));
+        report.print(translateStatus(this.trace.hasNoDefects()));
         report.print(" - ");
         report.print(this.trace.count());
         report.print(" total");
-        if (this.trace.countUncovered() != 0)
+        if (this.trace.countDefects() != 0)
         {
             report.print(", ");
-            report.print(this.trace.countUncovered());
-            report.print(" not covered");
+            report.print(this.trace.countDefects());
+            report.print(" defect");
         }
         report.print(this.newline);
     }
 
     private void renderFailureIds(final PrintStream report)
     {
-        this.trace.getUncoveredIds().stream() //
+        this.trace.getDefectIds().stream() //
                 .sorted()//
                 .forEachOrdered(id -> {
                     report.print(id);
                     report.print(this.newline.toString());
                 });
     }
 
-    private void renderFailureDetails(final PrintStream report)
+    private void renderFailureSummaries(final PrintStream report)
     {
-        this.trace.getUncoveredItems().stream() //
+        this.trace.getDefectItems().stream() //
                 .sorted(LINKED_ITEM_BY_ID) //
-                .forEachOrdered(item -> renderItemSummary(item, report));
+                .forEachOrdered(item -> renderItemSummary(report, item));
     }
 
-    // [impl->dsn~reporting.plain-text.specification-item-overview~1]
-    private void renderItemSummary(final LinkedSpecificationItem item, final PrintStream report)
+    // [impl->dsn~reporting.plain-text.specification-item-overview~2]
+    private void renderItemSummary(final PrintStream report, final LinkedSpecificationItem item)
     {
         report.print(translateStatus(!item.isDefect()));
         report.print(" - ");
-        renderItemLinkCounts(item, report);
+        renderItemLinkCounts(report, item);
         report.print(" - ");
         report.print(item.getId().toString());
         report.print(" ");
-        renderMaturity(item, report);
-        report.print(translateArtifactTypeCoverage(item));
+        renderMaturity(report, item);
+        report.print(translateArtifactTypeCoverage(report, item));
         report.print(this.newline);
     }
 
-    private String translateArtifactTypeCoverage(final LinkedSpecificationItem item)
+    private String translateArtifactTypeCoverage(final PrintStream report,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "report". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)